From: Bernd Waibel <waebbl-gentoo@posteo.net>
Date: Sat, 3 Jul 2021 22:52:29 +0200
Subject: [PATCH] find slotted openexr

Signed-off-by: Bernd Waibel <waebbl-gentoo@posteo.net>
---
 CMakeModules/FindOpenEXR.cmake | 11 +++++++++--
 1 file changed, 9 insertions(+), 2 deletions(-)

diff --git a/CMakeModules/FindOpenEXR.cmake b/CMakeModules/FindOpenEXR.cmake
index b5c3af3..bb930e6 100644
--- a/CMakeModules/FindOpenEXR.cmake
+++ b/CMakeModules/FindOpenEXR.cmake
@@ -19,14 +19,16 @@ else (OPENEXR_INCLUDE_DIR AND OPENEXR_LIBRARIES)
 IF (NOT WIN32 OR MINGW)
   # use pkg-config to get the directories and then use these values
   # in the FIND_PATH() and FIND_LIBRARY() calls
-  INCLUDE(UsePkgConfig)
+  INCLUDE(FindPkgConfig)
   
-  PKGCONFIG(OpenEXR _OpenEXRIncDir _OpenEXRLinkDir _OpenEXRLinkFlags _OpenEXRCflags)
+  PKG_CHECK_MODULES(OpenEXR REQUIRED OpenEXR-2)
+  #_OpenEXRIncDir _OpenEXRLinkDir _OpenEXRLinkFlags _OpenEXRCflags)
 ENDIF (NOT WIN32 OR MINGW)
   FIND_PATH(OPENEXR_INCLUDE_DIR ImfRgbaFile.h
      ${_OpenEXRIncDir}
      ${_OpenEXRIncDir}/OpenEXR/
      /usr/include
+     /usr/include/OpenEXR-2
      /usr/local/include
      ${SOURCE_BASE_DIR}/Deploy/include
      ${SOURCE_BASE_DIR}/Deploy/include/OpenEXR
@@ -38,6 +40,7 @@ ENDIF (NOT WIN32 OR MINGW)
     NAMES Half
     PATHS
       ${_OPENEXRLinkDir}
+      /usr/lib${LIB_SUFFIX}/OpenEXR-2
       ${SYSTEM_LIB_DIRS}
       ${SOURCE_BASE_DIR}/Deploy/lib/
       ${SOURCE_BASE_DIR}/Deploy/lib/Release
@@ -48,6 +51,7 @@ ENDIF (NOT WIN32 OR MINGW)
     NAMES Iex-2_2 Iex
     PATHS
       ${_OPENEXRLinkDir}
+      /usr/lib${LIB_SUFFIX}/OpenEXR-2
       ${SYSTEM_LIB_DIRS}
       ${SOURCE_BASE_DIR}/Deploy/lib/
       ${SOURCE_BASE_DIR}/Deploy/lib/Release
@@ -58,6 +62,7 @@ ENDIF (NOT WIN32 OR MINGW)
     NAMES IlmThread-2_2 IlmThread
     PATHS
       ${_OPENEXRLinkDir}
+      /usr/lib${LIB_SUFFIX}/OpenEXR-2
       ${SYSTEM_LIB_DIRS}
       ${SOURCE_BASE_DIR}/Deploy/lib/
       ${SOURCE_BASE_DIR}/Deploy/lib/Release
@@ -68,6 +73,7 @@ ENDIF (NOT WIN32 OR MINGW)
     NAMES Imath-2_2 Imath
     PATHS
       ${_OPENEXRLinkDir}
+      /usr/lib${LIB_SUFFIX}/OpenEXR-2
       ${SYSTEM_LIB_DIRS}
       ${SOURCE_BASE_DIR}/Deploy/lib/
       ${SOURCE_BASE_DIR}/Deploy/lib/Release
@@ -79,6 +85,7 @@ ENDIF (NOT WIN32 OR MINGW)
     NAMES IlmImf-2_2 IlmImf 
     PATHS
       ${_OPENEXRLinkDir}
+      /usr/lib${LIB_SUFFIX}/OpenEXR-2
       ${SYSTEM_LIB_DIRS}
       ${SOURCE_BASE_DIR}/Deploy/lib/
       ${SOURCE_BASE_DIR}/Deploy/lib/Release
-- 
2.32.0

